src/lib/connectors/shared/remote-debugging-connector.ts

```diff
@@ -488,7 +488,11 @@ export class Connector implements IConnector {
      * * uses `favicon.ico` and the final url after redirects.
      */
     private async getFavicon(element?: AsyncHTMLElement) {
-        const href = element ? element.getAttribute('href') : '/favicon.ico';
+        if (!element || !element.getAttribute('href')) {
```

Comment:
Also, I don't think `getFavicon` should be responsible of checking these things, and instead, and at this point:

https://github.com/sonarwhal/sonar/blob/941d439affca16e3af1b0df90e739ee746df2313/src/lib/connectors/shared/remote-debugging-connector.ts#L329-L331

all the checks should be made to determine if valid `favicon` was specified or not.